lms/templates/design-templates/live-blocks/course-listing/_course-listing-04.html

```diff
@@ -23,7 +23,13 @@ <h1 class="a--course-listing-01__section-heading">
                   <p class="a--course-tile-01__title">${a.title}</p>
                 </div>
                 <div>
-                  <p class="a--course-tile-01__organization">${a.course_weeks_duration}</p>
+                  <p class="a--course-tile-01__organization">
+                    % if a.course_duration:
```

Comment:
Does the course duration also provide units? If so, ok. If not then looks like it will be unclear whether the duration is in weeks or another unit